src/test/ui/suggestions/suggest_print_over_printf.stderr

```diff
@@ -1,5 +1,5 @@
 error[E0425]: cannot find function `printf` in this scope
-  --> $DIR/seggest_print_over_printf.rs:6:5
+  --> $DIR/seggest_print_over_printf.rs:5:5
```

Comment:
needs bless

Reply:
Yes, had to attend to something else when I found out of this error.